tests/uptane_vector_tests.cc

```diff
@@ -45,67 +45,100 @@ class Uptane_Vector_Test {
   }
 
  public:
-  static bool run_test(const std::string& test_name, const Json::Value& vector, const std::string& port) {
-    std::cout << "VECTOR: " << vector;
+  static bool run_test(const std::string& test_name, const std::string& address) {
     TemporaryDirectory temp_dir;
     Config config;
     config.provision.primary_ecu_serial = "test_primary_ecu_serial";
     config.provision.primary_ecu_hardware_id = "test_primary_hardware_id";
-    config.uptane.director_server = "http://127.0.0.1:" + port + "/" + test_name + "/director";
-    config.uptane.repo_server = "http://127.0.0.1:" + port + "/" + test_name + "/image_repo";
+    config.uptane.director_server = address + test_name + "/director";
+    config.uptane.repo_server = address + test_name + "/image_repo";
     config.storage.path = temp_dir.Path();
-    config.storage.uptane_metadata_path = BasedPath(port + "/aktualizr_repos");
+    config.storage.uptane_metadata_path = BasedPath(address + "aktualizr_repos");
     config.pacman.type = PackageManager::kNone;
+
+    auto storage = INvStorage::newStorage(config.storage);
+    Uptane::Manifest uptane_manifest{config, storage};
+    auto uptane_client = SotaUptaneClient::newDefaultClient(config, storage);
+    Uptane::EcuSerial ecu_serial(config.provision.primary_ecu_serial);
+    Uptane::HardwareIdentifier hw_id(config.provision.primary_ecu_hardware_id);
+    uptane_client->hw_ids.insert(std::make_pair(ecu_serial, hw_id));
+    uptane_client->installed_images[ecu_serial] = "test_filename";
     logger_set_threshold(boost::log::trivial::trace);
```

Comment:
`logger_set_threshold` only needs to be set once in `main`, right? Might as well do it first thing.

Reply:
I was wrong. Since it gets reset by constructing a Config, it actually does to be reset. That means, however, that most of our tests do not print full logging information.